earth_enterprise/BUILD.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
```

Comment:
I believe all our scripts and executables are lowercase. I think we should stick with that convention for these.